### PR TITLE
Add demo

### PR DESCRIPTION
**Pull request by meniRoy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sodu: false
+sudo: false
```
Comment (meniRoy) [inline .travis.yml]: test

File: projects/demo/src/app/main-window/main-window.component.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-main-window',
+  templateUrl: './main-window.component.html',
+  styleUrls: ['./main-window.component.scss']
```
Comment (alon6699) [inline projects/demo/src/app/main-window/main-window.component.ts]: unnecessary

File: projects/demo/src/app/main-window/main-window.component.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+})
+export class MainWindowComponent implements OnInit {
+
+  constructor() {
```
Comment (alon6699) [inline projects/demo/src/app/main-window/main-window.component.ts]: unnecessary

File: projects/demo/src/app/main-window/main-window.component.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  }
+
+  ngOnInit() {
```
Comment (alon6699) [inline projects/demo/src/app/main-window/main-window.component.ts]: unnecessary

File: projects/demo/src/app/new-window/new-window.component.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  selector: 'app-new-window',
+  templateUrl: './new-window.component.html',
+  styleUrls: ['./new-window.component.scss']
+})
+export class NewWindowComponent implements OnInit {
+
+  constructor(private electronService: ElectronService) {
+  }
+
+  ngOnInit() {
```
Comment (alon6699) [inline projects/demo/src/app/new-window/new-window.component.ts]: unnecessary

**Pull request merged**